protobuf/transform.go

```diff
@@ -167,6 +182,14 @@ func (t *Transformer) transformStruct(pkg *Package, s *scanner.Struct) *Message
 	return msg
 }
 
+func defaultOptionsForScannedMessage(s *scanner.Struct) Options {
+	opts := make(Options)
+
+	opts["(gogoproto.drop_type_declaration)"] = NewLiteralValue("true")
+
+	return opts
```

Comment:
I understand the necessity of having different `defaultOptionsForScanned*` functions. My suggestion was only regarding to the inner implementation of both.
I was thinking in something like

```
var dropDeclarationOption = Options{
     "(gogoproto.enum_drop_type_declaration)": NewLiteralValue("true"),
}
func defaultOptionsForScannedEnum(e *scanner.Enum) Options {
    return dropDeclarationOption
}
func defaultOptionsForScannedMessage(e *scanner.Struct) Options {
    return dropDeclarationOption
}
```

But I'm agree with the @erizocosmico proposal, and just return the inline created `Option{foo,bar}`